src/components/menu/menu.tsx

```diff
@@ -271,9 +271,10 @@ class MenuItem extends Component<any, any> {
   render(props: any): VNode<any> {
     const badgeType: string | null =
       props.data.badgeType && !props.isSelected(props.data) ? BadgeType[props.data.badgeType] : BadgeType[props.data.badgeType + 'Active'];
+      const isAdvanced = props?.data?.key == "adv_captions" ? true : false;
     return (
       <div
-        role="menuitemradio"
+        role={isAdvanced ? "menuitem" : "menuitemradio"}
```

Comment:
Use const values for this menuitem & menuitemradio

Reply:
done